docs/platform/release-notes/tizen-5-5-m2.md

```diff
@@ -0,0 +1,641 @@
+    - Support for alerts from 4396 ~ 4399 has been added to message API set.
+    - Support to distinguish CMAS public safety alerts, public safety for an additional language has been added to message service.
+    - Public safety alerts for additional languages have been added to message service.
+
+### Fixes
+
+- Maps
+    - Dependency on deprecated packages has been removed.
+
+
+## Web framework
+
+### New and changed features
+
+- Tizen Web Engine
+  - Chromium open source version has been changed.
+    - Chromium-efl base open source version has been changed from M63 to M69. 
+    - Distributed web engine feature has been added.
+This feature allows distributed web contents processing on multi device via IP network connectivity.
+- Web Runtime
+  - Web Runtime new feature has been applied.
+    - WRT Service application feature has been added.
+: This feature allow application developer write service app with javascript
+    - Support WRT Add-on has been added.
+: This feature allows 3rd party developer extend WRT feature or customize WRT policy /w extension module which is written with javascript.
+- Web UI Framework
+  - New TAU UI component has been added.
+    - Multi shape graph component, contents carousel, 3D container
+  - WYSIWYG UI build has been added for Easy Web application development.
+
+
+## Lightweight Web Solution
+
+### New and changed features
+
+- Rendering Engine
+  - Canvas element and its properties have been added.
+  - SVG elements and their properties have been extended.
+  - CSS animation feature has been added.
+  - WebSocket feature has been added.
+  - Animated GIF has been supported.
+- JS Engine (ES6 Features)
+  - Class type has been added.
+  - Pattern feature has been added.
+  - Proxy feature has been added.
+  - Iterator Operations have been added.
+  - Extended Parameter has been added.
+  - Generator has been added.
+- Support for headless profile has been added.
+- Performance and Memory Optimization
+  - Rendering Engine
+    - Direct mode has been enabled on EvasGL backend.
+    - CompositorGL has been fixed to enhance performance.
+    - Multithreaded image decoding feature has been added.
+    - OpenGL Texture is re-used to reduce loading time.
+  - JS Engine
+    - Lexer's objects are allocated on stack memory rather than heap memory.
+    - Interpreter's initialization process has been simplified.
+
+### Fixes
+
+- Bug fixes
+  - Layout bugs have been fixed.
+  - Memory leaks have been fixed.
+  - Crashes have been fixed.
+
+
+## Tizen .NET
+
+### New and changed features
+
+- .NET Core (Runtime)
+  - .NETCore has been upgraded from 3.0.0-preview2 to 3.0.0.
+- Xamarin.Forms
+  - Xamarin.Forms 4.2.0, which is latest stable version has been supported.
+- Tizen.CircularUI
+  - Wearable UI extension (Tizen.Wearable.CircularUI) 1.3.0 has been supported.
+- TizenFX (C# Device API) API Level 6 
+  - Deprecated API set of OAuth2 which were uncovered has been removed.
```

Comment:
Deprecated API sets of OAuth2, which were uncovered has been removed.

Reply:
This item should be removed.